staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -560,6 +560,20 @@ type DeleteOptions struct {
 	// +optional
 	// +listType=atomic
 	DryRun []string `json:"dryRun,omitempty" protobuf:"bytes,5,rep,name=dryRun"`
+
+	// IgnoreStoreReadErrorWithClusterBreakingPotential, if set to true,
+	// will trigger an unsafe deletion of the resource in case the
+	// normal deletion flow fails with a corrupt object error.
+	// A resource is considered corrupt if it can not be retrieved from
+	// the underlying storage because of a) its data can not be transformed
```

Comment:
"transform" is not a term in the API, is it? At least I would mention "e.g. decrypted ....".